tests/integ/test_session.py

```diff
@@ -46,5 +46,5 @@ def test_sagemaker_session_does_not_create_bucket_on_init(
         default_bucket=CUSTOM_BUCKET_NAME,
     )
 
-    s3 = boto3.resource("s3")
+    s3 = boto3.resource("s3", region_name=DEFAULT_REGION)
```

Comment:
should this be hardcoded? (I also recognize that this applies more broadly to the file, and can happen in a separate PR.)

Reply:
Fixed in https://github.com/aws/sagemaker-python-sdk/pull/1381